.env.production

```diff
@@ -0,0 +1,11 @@
+RECORD_MANAGER_API_URL=http://localhost:1235/record-manager/services/record-manager-server
+RECORD_MANAGER_APP_TITLE=Record Manager
+RECORD_MANAGER_APP_INFO="blascom@gmail.com"
```

Comment:
Neni dobry vnorovat achor tady do html, protoze to budu muset udelat pres dangerouslySetInnerHtml (coz neni bezpecny, protoze je tam riziko XSS utoku). 
Kde bych mel rozmistit "© KBSS at FEE CTU in Prague, 2024" ?

Reply:
OK, good point, I suggest putting there only text defined in APP_INFO

Reply:
So the text could be "© KBSS at FEE CTU in Prague, 2024". Leave default blank.

Reply:
eI suggest two alternative solutions :
1) do not include link at all
2) include link in the environment variable  RECORD_MANAGER_APP_INFO_LINK. It may also be empty which would cause no <a href ..> beeing rendered.

Of course I prefer 2), but have no problem with 1) at all.

Reply:
I would like to save the _RECORD_MANAGER_APP_INFO_LINK_ with the link, to use it in the dockerFile, and use some other env variable for the client code(e.g just _RECORD_MANAGER_APP_INFO_), which will not contain any html tags, but will contain the actual link, and then in the client code in _JSX_ I will have an _achor_ tag, and will just use that env variable as _href_. What do you think ? 

Otherwise I would like to avoid using dangerouslySetInnerHtml, but if for now it is more important, we can continue with the 2 option

Reply:
Actual state is on the image, I used RECORD_MANAGER_APP_INFO, but it is empty in .env file, then in config/index.js I set the default value for this variable with the given text. Also, I've squashed last couple of commits(that solve this). Please let me know if it is ok 
<img width="862" alt="Screenshot 2024-02-29 at 10 21 59" src="https://github.com/akaene/record-manager-ui/assets/50382323/0adbe1c7-49c9-4a8c-9f23-93b10e5b6dab">